en/core-api/credit-card.md

```diff
@@ -396,13 +387,13 @@ public class MidtransExample {
 ```
 
 #### **Python**
-For Python, install [**midtrans-client**](https://github.com/Midtrans/midtrans-python-client) PIP package
+
+Install [**midtransclient**](https://github.com/Midtrans/midtrans-python-client) PIP package
 ```bash
 pip install midtransclient
 ```
 
-**Sample Request**
-
+Card Transaction Charge
 ```python
```

Comment:
Other inconsistency is now fixed. 
But you also seems to introduce another inconsistency here, why is this **Python tabs** now says `Card Transaction Charge` while all the other tabs are `**Sample Request**`?
Whichever is fine, just be consistent with it please.